### PR TITLE
fcos-projects: Link to Podman Machine

### PR DESCRIPTION
**Pull request by cgwalters**
Split out this not-controversial part from
https://github.com/coreos/fedora-coreos-docs/pull/515

### CODE REVIEW

File: modules/ROOT/pages/fcos-projects.adoc [hunk 1]
```diff
@@ -2,6 +2,7 @@
 
 This a list of projects that are actively using Fedora CoreOS:
 
+* https://docs.podman.io/en/latest/markdown/podman-machine.1.html[Podman Machine] uses Fedora CoreOS to run containers in a local environment, and also has a `podman machine os apply` command that allows customization.
```
Comment (jlebon) [inline modules/ROOT/pages/fcos-projects.adoc:5]: Might be worth mentioning here that it's available on non-Linux.

**Review by jlebon — APPROVED** [review]

**Review by dustymabe — APPROVED** [review]

File: modules/ROOT/pages/fcos-projects.adoc (hunk 1 above)
Comment (cgwalters) [reply modules/ROOT/pages/fcos-projects.adoc]: Done

**Review by jlebon — APPROVED** [review]

File: modules/ROOT/pages/fcos-projects.adoc [hunk 2]
```diff
@@ -2,6 +2,7 @@
 
 This a list of projects that are actively using Fedora CoreOS:
 
+* https://docs.podman.io/en/latest/markdown/podman-machine.1.html[Podman Machine] uses Fedora CoreOS to run containers in a local environment (notably including Windows and MacOS X), and also has a https://docs.podman.io/en/latest/markdown/podman-machine-os.1.html[podman machine os] command that allows customization of Fedora CoreOS in a container-native way.
```
Comment (bgilbert) [inline modules/ROOT/pages/fcos-projects.adoc:5]: Nit:
```suggestion
* https://docs.podman.io/en/latest/markdown/podman-machine.1.html[Podman Machine] uses Fedora CoreOS to run containers in a local environment (notably including Windows and macOS), and also has a https://docs.podman.io/en/latest/markdown/podman-machine-os.1.html[podman machine os] command that allows customization of Fedora CoreOS in a container-native way.
```

**Review by bgilbert — COMMENTED** [review]
LGTM

File: modules/ROOT/pages/fcos-projects.adoc (hunk 2 above)
Comment (bgilbert) [reply modules/ROOT/pages/fcos-projects.adoc:5]: Fix in #523.

**Pull request merged**